cypress/integration/gift_card_spec.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference types="Cypress" />
+import { HomePo } from '../support/home.po';
+import { GiftCardsPo } from '../support/giftCard.po';
+
+
+describe('100$ gift card add to cart', () => {
+    const homePage = new HomePo();
+    const giftPage = new GiftCardsPo();
+
+    beforeEach(() => cy.visit(homePage.pageUrl));
+
+    it('gift card add to cart', () => {
+
+        const successMessage = 'Added to Cart';
+        cy.viewport(1175, 600);
+
+        giftPage.clickOnElem(giftPage.giftCardLink);
+        giftPage.isURLcontain(giftPage.giftCardUrl);
+        giftPage.clickOnElem(giftPage.eGift);
+        giftPage.isElemVisible(giftPage.prise100);
+        giftPage.clickOnElem(giftPage.prise100);
```

Comment:
Then this item becomes selected - this check still not covered, need to add smth like this:
cy.get('@price').parents('.a-button.a-button-toggle').should('to.have.class', 'a-button-selected');